packages/playground/src/components/manage_gateway_dialog.vue

```diff
@@ -262,30 +268,61 @@ export default {
     const selectedNode = ref();
 
     watch(selectedK8SNodeName, getSupportedNetworks, { deep: true });
+    const tableHeaders = ref([
+      { title: "Name", key: "name" },
+      { title: "Contract ID", key: "contractId" },
+      { title: "Domain", key: "domain" },
+      { title: "TLS Passthrough", key: "tls_passthrough" },
+      { title: "Backend", key: "backends", sortable: false },
+      { title: "Status", key: "status" },
+      { title: "Actions", key: "actions", sortable: false },
+    ]);
 
     onMounted(async () => {
       updateGrid(grid, { projectName: "" });
       suggestName();
       await loadGateways();
       getSupportedNetworks();
+      updateHeaders();
     });
 
-    const tableHeaders = [
-      { title: "Name", key: "name" },
-      { title: "Contract ID", key: "contractId" },
-      { title: "Domain", key: "domain" },
-      { title: "TLS Passthrough", key: "tls_passthrough" },
-      { title: "Backend", key: "backends" },
-      { title: "Status", key: "status" },
-      { title: "Actions", key: "actions" },
-    ];
+    const updateHeaders = () => {
+      if (props.k8s) {
+        const actionsIndex = tableHeaders.value.findIndex(header => header.key === "actions");
+        if (actionsIndex !== -1) {
+          tableHeaders.value.splice(actionsIndex, 0, { title: "Attached to", key: "attached_to", sortable: false });
+        }
+      }
+    };
+
+    const getDomainNode = (domain: GridGateway): string => {
+      const getIP = (url: string) => url.split("://")[1].split(":")[0];
+      const IP = getIP(domain.backends[0]);
```

Comment:
Use URL constructor instead of parsing url manually
```ts
const getIP = (url) => new URL(url).hostname

getIP1('https://1.1.1.1:8000') // '1.1.1.1'
```

Reply:
Resolved